CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - `Campaign` no longer allows overlapping names between parameters and targets
+- Random seed context is correctly set within benchmarks.
```

Comment:
Do I also ned to add anything for contributors file or can I leave it as it is since it was a bugfix and not a feature?